.github/workflows/compile.js.yml

```diff
@@ -131,7 +133,8 @@ jobs:
           npm run build-storybook
 
       - name: Publish to Chromatic
-        if: ${{ env.HAVE_CHROMATIC_PROJECT_TOKEN == 'true' && env.IS_CHROMATIC_DISABLED == 'false'}}
+        if: ${{ env.HAVE_CHROMATIC_PROJECT_TOKEN == 'true' && env.IS_CHROMATIC_DISABLED == 'false' && 
+          (github.event_name == 'workflow_dispatch' || (github.event_name == 'pull_request' && github.event.action == 'opened')) }}
```

Comment:
if its a pull request, it should run on all commits in the pull request, not just the open tag (i.e. if you do 'fixes' it won't show the updated chromatic docs and delta image diff)

Reply:
It also does not run on version cuts nor primary branches which should always run.